inc/Engine/Optimization/DeferJS/DeferJS.php

```diff
@@ -289,4 +274,36 @@ public function exclude_jquery_upgrade() {
 
 		update_option( 'wp_rocket_settings', $options );
 	}
+
+	/**
+	 * Get exclusion list pattern.
+	 *
+	 * @return string
+	 */
+	private function get_inline_exclusions_list_pattern() {
+		$inline_exclusions_list = $this->inline_exclusions;
+
+		/**
+		 * Filters the patterns used to find inline JS that should not be deferred
+		 *
+		 * @since 3.8
+		 *
+		 * @param array $inline_exclusions_list Array of inline JS that should not be deferred.
+		 */
+		$additional_inline_exclusions_list = apply_filters( 'rocket_defer_inline_exclusions', null );
+
+		$inline_exclusions = '';
+
+		if ( is_string( $additional_inline_exclusions_list ) ) {
+			$additional_inline_exclusions_list = explode( '|', $additional_inline_exclusions_list );
+		}
+
+		$inline_exclusions_list = array_merge( $inline_exclusions_list, (array) $additional_inline_exclusions_list );
+
+		foreach ( (array) $inline_exclusions_list as $inline_exclusions_item ) {
```

Comment:
I don't think type casted in there is needed.
`array_merge` function always return an array